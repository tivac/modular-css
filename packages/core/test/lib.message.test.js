"use strict";

var message = require("../lib/message.js");

describe("/lib", function() {
    describe("/message.js", function() {
        it("should find the last message containing a given field", function() {
            expect(
                message({ messages : [
                    { fooga : { aooga : "aooga" } },
                    { fooga : { booga : "booga" } },
                    { fooga : { cooga : "cooga" } }
                ] }, "fooga")
            )
            .toMatchSnapshot();
        });
        
        it("should return an empty object if a message wasn't found", function() {
            expect(
                message({ messages : [
                    { fooga : { aooga : "aooga" } },
                    { fooga : { booga : "booga" } },
                    { fooga : { cooga : "cooga" } }
                ] }, "booga")
            )
            .toMatchSnapshot();
        });
    });
});
