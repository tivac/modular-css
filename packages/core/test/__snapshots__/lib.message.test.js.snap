// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/lib /message.js should find the last message containing a given field 1`] = `
Object {
  "cooga": "cooga",
}
`;

exports[`/lib /message.js should return an empty object if a message wasn't found 1`] = `Object {}`;
